applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -696,7 +716,14 @@ static void send_report_mouse(u8_t report_id, struct report_data *rd)
 
 	EVENT_SUBMIT(event);
 
-	rd->items.update_needed = false;
+	if ((rd->axes.axis[0] != 0) ||
+	    (rd->axes.axis[1] != 0) ||
+	    (rd->axes.axis[2]  < -1) || (rd->axes.axis[2]  > 1)) {
+		/* If there is some axis data to send, request report update. */
+		rd->items.update_needed = true;
```

Comment:
Maybe we can simplify it to one `update_needed` per report data?

Reply:
Currently we do not use the update_needed from `rd->axes`

Reply:
will do. thanks, overlooked.

Reply:
done